base/src/main/java/uk/ac/ebi/atlas/solr/analytics/baseline/BaselineAnalyticsSearchDao.java

```diff
@@ -3,18 +3,29 @@
 import com.jayway.jsonpath.Configuration;
 import com.jayway.jsonpath.JsonPath;
 import com.jayway.jsonpath.Option;
+import org.apache.commons.math.util.MathUtils;
 import org.springframework.stereotype.Component;
-import uk.ac.ebi.atlas.profiles.baseline.BaselineExpressionLevelRounder;
 import uk.ac.ebi.atlas.search.SemanticQuery;
 import uk.ac.ebi.atlas.solr.analytics.query.AnalyticsQueryClient;
 
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.function.DoubleUnaryOperator;
 
 @Component
 public class BaselineAnalyticsSearchDao {
-    // analyticsQueryClient.baselineFacets searches for baseline experiments
+    static final int MIN_PRECISION = 7;
+    static final DoubleUnaryOperator BASELINE_EXPRESSION_ROUNDER =
```

Comment:
Talking about style guides, isn't all caps snake case more commonly used for constants? I would argue that this rounder is a bit more than a constant...

Reply:
`static final` means it’s a constant. Don’t let types deceive you! ;)